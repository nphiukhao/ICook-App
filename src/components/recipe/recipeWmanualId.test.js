
import React from 'react'
import RecipeManualId from './recipeWmanualId'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`RecipeWManualId component`, () => {
    it('RecipeManualId snapshot test', () => {
        const tree = shallow(<RecipeManualId/>)
        expect(toJson(tree)).toMatchSnapshot()
    })
})
