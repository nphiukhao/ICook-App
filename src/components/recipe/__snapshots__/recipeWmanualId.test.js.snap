// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeWManualId component RecipeManualId snapshot test 1`] = `""`;
